.gitignore

```diff
@@ -20,7 +20,7 @@
 /go-engine/unleash_engine.h
 /php-engine/unleash_engine.h
 /php-engine/libyggdrasilffi.so
```

Comment:
Should we also change this for php-engine?

Reply:
Honestly I'm not sure yet. I'm don't yet understand how PHP distributes native binaries or whether or not these tricks will be needed there but definitely in a future PR